lighthouse-core/report/html/report-styles.css

```diff
@@ -35,6 +35,7 @@
 .lh-vars {
   /* Palette using Material Design Colors
    * https://www.materialui.co/colors */
+  --color-amber: #FFF8E1; /* 50 */
```

Comment:
not sure what 50 means

Reply:
yeah I was confused too until I looked up the material ui table :) https://www.materialui.co/colors

Reply:
oh, in that case, should be part of the name. 

> Omit "-{intensity}" if there is only one color of that hue.

isn't simplifying anything if it's helpful enough to have the intensity that we add it in a comment :)

Reply:
done